src/main.rs

```diff
@@ -117,6 +117,11 @@ struct Cli {
     #[arg(long)]
     /// If true, will load the locally compiled system contracts (useful when doing changes to system contracts or bootloader)
     dev_use_local_contracts: bool,
+
+    #[arg(long)]
+    /// Whether to override the l1 gas price.
+    /// If not set - fork or default value(`1 gwei`) will be used.
+    override_l1_gas_price: Option<u64>,
```

Comment:
Can we avoid having another CLI argument in favor of a new `RPC` endpoint in the `CONFIG` namespace? e.g. `config_setL1GasPrice(gasPrice: u64)`

The idea would be to have the test node configuration editable without restarting the node.

Example (Would go in https://github.com/matter-labs/era-test-node/blob/main/src/configuration_api.rs):
```rust
fn config_set_l1_gas_price(&self, gas_price: u64) -> Result<u64> {
    let mut inner = self.node.write().unwrap(); // We're cleaning up `.unwrap()` use, just leaving here for simplicity
    inner.l1_gas_price = gas_price;
    Ok(inner.l1_gas_price)
}
```

Reply:
Have not seen it, but sounds good. Will take a look, thanks